### ISSUE TITLE
Omit significance stars

### ISSUE THREAD

**Issue by bwiernik** [dstanley4/apaTables#26]
Is there a way to omit significance stars and show only confidence intervals instead? I don’t report p values or stars in my work.

**Comment by dstanley4** [comment]
Yes - this approach is consistent with new ASA guidelines. Development version now includes show.sig.stars (BOOL) for correlation table. More general implementation for other functions will follow as do more general updates to each function.
